libr/util/str.c

```diff
@@ -631,7 +631,7 @@ R_API const char *r_str_rsep(const char *base, const char *p, const char *sep) {
 	return NULL;
 }
 
-R_API const char *r_str_rstr(const char *base, const char *p) {
+R_API char *r_str_rstr(const char *base, const char *p) {
```

Comment:
uhm, nope input is const, so output must be const

Reply:
ok 2f842891fae5d3a4310c7a70c80ecde45e01ecd4